pghoard/transfer.py

```diff
@@ -58,8 +67,31 @@ def form_key_path(file_to_transfer):
                             file_to_transfer["filetype"],
                             name)
 
+    def transmit_statsd_metrics(self):
+        """
+        Keep statsd updated about how long time ago each filetype was successfully uploaded.
+        Transmits max once per ten seconds, regardless of how many threads are running.
+        """
+        with _STATS_LOCK:
+            if time.time() - self._last_stats_transmit_time < 10.0:
```

Comment:
isn't `self._last_stats_transmit_time` a value local to this thread - ie every thread can transmit once per 10 seconds?